src/runtime/virtcontainers/qemu_ppc64le.go

```diff
@@ -115,10 +115,6 @@ func (q *qemuPPC64le) bridges(number uint32) {
 	q.Bridges = genericBridges(number, q.qemuMachine.Type)
 }
 
-func (q *qemuPPC64le) cpuModel() string {
-	return defaultCPUModel
-}
-
```

Comment:
This change makes sense but it isn't really related to this PR either.

Reply:
Same here. It's simple enough to go through in the same PR but in a separate commit.